README.md

```diff
@@ -108,13 +77,7 @@ After you're finished please remove all the comments and instructions!
 
```

Comment:
## 1st Getting Started Section
![image](https://user-images.githubusercontent.com/93039359/224508542-4c7a5138-5282-4086-a679-09c3a37441c6.png)

## 2nd Getting Started Section
![image](https://user-images.githubusercontent.com/93039359/224508645-98d0bcc6-ef0f-4a1d-ba61-9f50b8843e14.png)

**[OPTIONAL]**

- [ ] Kindly move 1st Getting Started section shown in the image above in place of the 2nd Getting Started Section 👍 to make your readme file descriptive and professional 😊